### DISCUSSION TITLE
How Can I get Sponsored?

### DISCUSSION THREAD

**Discussion by barandenizdogan** [community/community#40045] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am new here and You can check my sponsor page. Don't have much time to coding but I am always improving myself to do it better. How can I get sponsored as a student?

https://github.com/sponsors/barandenizdogan

**Comment by MitkoVtori** [comment]
### You already have the sponsor think:
![Screenshot 2022-11-25 101146](https://user-images.githubusercontent.com/112943652/203932756-d6977638-85bf-4d01-997c-3ccee2602571.png)
#### Just share the link so more people can see and sponsor.

If you want something like a student pack, because I saw you said "sponsored as a student", You can check this out: https://education.github.com/pack

**Reply by barandenizdogan** [reply]
Thanks for response. I already use the Github education pack but thanks for your advice.
